test/e2e/performanceprofile/functests/1_performance/netqueues.go

```diff
@@ -86,14 +83,8 @@ var _ = Describe("[ref_id: 40307][pao]Resizing Network Queues", Ordered, Label(s
 
 	AfterEach(func() {
 		By("Reverting the Profile")
-		spec, err := json.Marshal(initialProfile.Spec)
-		Expect(err).ToNot(HaveOccurred())
-		Expect(testclient.Client.Patch(context.TODO(), profile,
-			client.RawPatch(
-				types.JSONPatchType,
-				[]byte(fmt.Sprintf(`[{ "op": "replace", "path": "/spec", "value": %s }]`, spec)),
-			),
-		)).ToNot(HaveOccurred())
+		profile.Spec = initialProfile.Spec
+		testclient.ControlPlaneClient.Update(context.TODO(), profile)
```

Comment:
likewise, this should be wrapped in a `Eventually` block (or a retry loop in general)